cyences_app_for_splunk/default/data/ui/views/cs_o365_reports.xml

```diff
@@ -53,6 +60,60 @@
       </table>
     </panel>
   </row>
+  <row>
+    <panel>
+      <title>Azure - Current Security Score(In Percentage)</title>
+      <single>
+        <search>
+          <query>`cs_azure_securityscore` sourcetype="graphsecurity:score"  $tkn_host$ | eval percentage = (currentScore/maxScore)*100 | timechart max(percentage) as percentage | streamstats last(percentage) as percentage</query>
```

Comment:
Bro, we should use latest(percentage) instead of max. As we are looking for latest value of security score, there is chance of security score going down.